spatialpandas/io/parquet.py

```diff
@@ -98,20 +99,25 @@ def _get_geometry_columns(pandas_metadata):
 
 def to_parquet(
     df: GeoDataFrame,
-    fname: PathType,
+    path: PathType,
     compression: Optional[str] = "snappy",
     index: Optional[bool] = None,
+    storage_options: Optional[Dict[str, Any]] = None,
     **kwargs: Any,
 ) -> None:
     # Standard pandas to_parquet with pyarrow engine
-    pd_to_parquet(
-        df,
-        fname,
-        engine="pyarrow",
-        compression=compression,
-        index=index,
-        **kwargs,
-    )
+    to_parquet_args = {
+        "df": df,
+        "path": path,
+        "engine": "pyarrow",
+        "compression": compression,
+        "index": index,
+    }
+
+    if PANDAS_GE_12:
+        to_parquet_args.update({"storage_options":storage_options})
+
+    pd_to_parquet(**to_parquet_args, **kwargs)
```

Comment:
```suggestion
    to_parquet_args = {
        "df": df,
        "path": path,
        "engine": "pyarrow",
        "compression": compression,
        "index": index,
        **kwargs,
    }

    if PANDAS_GE_12:
        to_parquet_args.update({"storage_options": storage_options})
    else:
        to_parquet_args.update(storage_options)

    pd_to_parquet(**to_parquet_args)
```

Reply:
Do we need to the `else` block?

Reply:
My thought was that since pandas passes other kwargs through to the engine, this basically allows those options to flow through that way.

Reply:
After discussion, the purpose of this change was better addressed by passing filesystem instead.